packages/shared_preferences/shared_preferences_platform_interface/lib/method_channel_shared_preferences.dart

```diff
@@ -45,12 +45,12 @@ class MethodChannelSharedPreferencesStore
   }
 
   @override
-  Future<bool> clear() {
+  Future<bool?> clear() {
```

Comment:
The return type of `invokeMethod` is `Future<T?>`, that is why the return type of `clear` is `Future<bool?>`.